code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -178,6 +178,15 @@
 	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "chemical_laughter", /datum/mood_event/chemical_laughter)
```

Comment:
you might wanna change your changelog/PR description a bit, as laughter already HAS positive mood effects (albeit only if it gets into your system). however, I think that giving it extra mood effects when used on slipped people (as you've done in this PR) is still a great idea.

you might want to move the laughing and the chemical laughter over to expose_mob instead of on_mob_life, so that splashing laughter on someone won't do the slip mood event stuff without making them laugh.